packages/component/src/Styles/useInjectCustomProperties.ts

```diff
@@ -0,0 +1,58 @@
+import { hooks } from 'botframework-webchat-api';
+import { makeCreateStyles } from 'botframework-webchat-styles';
+import { useMemo } from 'react';
+import CustomPropertyNames from './CustomPropertyNames';
+import useInjectStyles from '../hooks/internal/useInjectStyles';
+import useNonce from '../hooks/internal/useNonce';
+
+const { useStyleOptions } = hooks;
+
+const webchatCustomPropertiesClass = 'webchat__css-custom-properties';
+
+export default function useInjectCustomProperties() {
+  const [styleOptions] = useStyleOptions();
+  const nonce = useNonce();
+  const [styles, className] = useMemo(() => {
+    const {
+      accent,
+      borderAnimationColor1,
+      borderAnimationColor2,
+      borderAnimationColor3,
+      bubbleMaxWidth,
+      bubbleMinHeight,
+      fontSizeSmall,
+      markdownExternalLinkIconImage,
+      paddingRegular,
+      primaryFont,
+      subtle,
+      timestampColor
+    } = styleOptions;
+
+    // eslint-disable-next-line no-magic-numbers
+    const randomClass = `wc-${Math.ceil(Math.random() * Number.MAX_SAFE_INTEGER).toString(36)}`;
```

Comment:
We can't use `Math.random()` despite we have no crypto concern here.

Need to use `math-random` or anything under Web Crypto API.